src/main/java/org/tugraz/sysds/parser/DataExpression.java

```diff
@@ -109,31 +113,70 @@
 	
 	public static final String DELIM_SPARSE = "sparse";  // applicable only for write
 	
-	public static final String[] RAND_VALID_PARAM_NAMES = 
-		{RAND_ROWS, RAND_COLS, RAND_DIMS, RAND_MIN, RAND_MAX, RAND_SPARSITY, RAND_SEED, RAND_PDF, RAND_LAMBDA};
+	public static final Set<String> RAND_VALID_PARAM_NAMES = 
```

Comment:
If this is a formatting change, the progress might be slowed. The best method would be to make any formatting changes in a different PR. or adhering to the existing formatting.

Reply:
I needed to add privacy to the parameter array and then I noticed inconsistencies in how parameters are checked using the arrays, so I decided to change it. It could easily be a separate PR and I will try to be better at making this kind of update in separate PRs in the future.